test/arithmetic/floating_point/floating_point_sqrt_test.dart

```diff
@@ -0,0 +1,200 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Indentifier: BSD-3-Clause
+//
+// floating_point_sqrt.dart
+// An abstract base class defining the API for floating-point square root.
+//
+// 2025 March 5
+// Authors: James Farwell <james.c.farwell@intel.com>,
```

Comment:
nit:  Align your names to get top billing -- like to see your names clearly.

Reply:
Done